release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md

```diff
@@ -68,8 +68,16 @@ A few backward-incompatible changes have been introduced during the Swan Lake Be
 
 #### New features
 
+##### Compiler API
+
+- Introduced a set of builders in the Types API, which are used to construct complex types that have varying components
```

Comment:
Are these usually listed under language changes? We don't expect the average Ballerina user to know of the semantic API, right?

Reply:
Ah, nice catch. This should go under `Developer Tools`. Shall we change this @sanjana ?

Reply:
Will be fixed in #2094